src/components/buttons.jsx

```diff
@@ -9,41 +9,43 @@ const Buttons = ({ initialStart, isRunning, start, stop, reset, addLap }) => {
   let lapResetButtonDisabled;
```

Comment:
Apologies, there is no direct feedback on this file. My formatter went to town on it though >.<